v3/src/components/graph/adornments/count/count-adornment-component.tsx

```diff
@@ -50,32 +58,87 @@ export const CountAdornment = observer(function CountAdornment({ model, cellKey,
     }
   }, [defaultFontSize, fontSize, graphModel.adornmentsStore, plotWidth, textContent])
 
+  const plotCaseCounts = useCallback(() => {
+    // If there are movable values present, we need to show the case count within each sub-plot region defined by the
+    // movable values and the min and max of the primary axis. For example, if there is one movable value, the sub-plot
+    // will have two regions, one from the axis' min value to the movable value, and another from the movable value
+    // to the axis' max value.
+
+    if (subPlotRegionBoundaries.current.length < 3 || graphModel.plotType !== "dotPlot") {
+      // If there are no movable values present, we just show a single case count.
+      displayCount.current = <div>{textContent}</div>
+      return
+    }
+
+    const regionCountParams = {
```

Comment:
```suggestion
    const regionCountParams: IRegionCountParams = {
```
Declaring its type will allow it to by type-checked at creation time.